debian/control

```diff
@@ -15,21 +15,38 @@ Standards-Version: 4.5.1
 
 Package: pimodbus-master
 Architecture: any
+Section: oldlibs
+Depends:
+  revpi-modbus-client,
+  ${misc:Depends}
+Description: transitional package
+  This is a transitional package. It can safely be removed.
```

Comment:
```suggestion
Description: transitional renaming package to revpi-modbus-client
 This is a transitional package. It can safely be removed.
```
The default indentation of this file for block texts is one space. We have introduced the two spaces at Depends for readability.

Reply:
I already fixed this earlier. Sorry for the inconvenience.

Reply:
The identation was already fixed earlier. I also changed the brief description.